projects/Mallard/src/components/front/card-group/card.tsx

```diff
@@ -5,15 +5,16 @@ import { withNavigation, NavigationInjectedProps } from 'react-navigation'
 import { Highlight } from '../../highlight'
 
 import { useArticleAppearance } from '../../../theme/appearance'
-import { FrontArticle } from '../../../common'
+import { Article } from '../../../common'
```

Comment:
I think an absolute import might work best here to avoid ever accidentally getting it from backend.